aleph/views/context.py

```diff
@@ -168,41 +172,42 @@ def finalize_response(resp):
     return resp
 
 
+def setup_logging_context(request):
+    role_id = None
+    if hasattr(request, "authz"):
+        role_id = request.authz.id
+    # Set up context varibales for structured logging
+    clear_contextvars()
+    bind_contextvars(
```

Comment:
Does this mean that if a request doesn't result in a log message, it will not end up in BigQuery? I also don't think we need all that data in each log message necessarily (although it probably doesn't hurt).